### PR TITLE
fix: fix job status

### PR DESCRIPTION
**Pull request by rephus**
Closes: 

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/models/SchedulerModel/index.ts [hunk 1]
```diff
@@ -454,7 +454,12 @@ export class SchedulerModel {
                 logDb.target_type === null
                     ? undefined
                     : (logDb.target_type as SchedulerLog['targetType']),
-            details: logDb.details === null ? undefined : logDb.details,
+            details: {
+                projectUuid: 'missing-project-uuid',
```
Comment (rephus) [inline packages/backend/src/models/SchedulerModel/index.ts:458]: edge case, old jobs will not be backwards compatible

**Comment by rephus** [comment]
I need to review this api test

> should replace project embed configuration

**Review by ZeRego — APPROVED** [review]

**Comment by rephus** [comment]
api tests are fixed after merging main
